devito/core/gpu.py

```diff
@@ -120,17 +120,17 @@ def _rcompile_wrapper(cls, **kwargs0):
         options0 = kwargs0.pop('options')
 
         def wrapper(expressions, mode='default', options=None, **kwargs1):
-            options = {**options0, **(options or {})}
             kwargs = {**kwargs0, **kwargs1}
 
             if mode == 'host':
-                par_disabled = options['par-disabled']
+                options = options or {}
```

Comment:
Could you set the kwarg default to `options={}` rather than `options=None`, thereby removing this line, and simplifying line 133 below?

Reply:
setting kwargs to mutable objects is strongly discouraged as it potentially leads to subtle bugs, so we just never do it